crates/extrinsics/src/lib.rs

```diff
@@ -183,7 +182,8 @@ where
         .build();
     let mut tx = client
         .tx()
-        .create_signed_offline(call, signer, params.into())?
+        .create_partial_offline(call, params.into())?
```

Comment:
Slight change in API introduced in https://github.com/paritytech/subxt/commit/b6b9ac65c76f2ad8246b4f55283d81be2c2ec000